nilearn/plotting/img_plotting.py

```diff
@@ -957,20 +957,26 @@ def plot_glass_brain(stat_map_img,
         Frontal, Axial, and Lateral). The brain glass schematics
         are added on top of the image.
 
+        The plotted image should be in MNI space for this function to work
+        properly.
+
         Parameters
         ----------
         stat_map_img : Niimg-like object
             See http://nilearn.github.io/manipulating_images/manipulating_images.html#niimg.
             The statistical map image. It needs to be in MNI space
             in order to align with the brain schematics.
-        output_file : string, or None, optional
+        output_file : string, or None, optional.
```

Comment:
I wouldn't add a dot at the end since that's not what is done for other parameters